### PR TITLE
Contribute Modal Done

### PR DESCRIPTION
**Pull request by Rohitdhende**

### CODE REVIEW

**Review by anitsingh367 — COMMENTED** [review]
kindly do the changes

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "react-toastify": "^9.0.8",
```
[the remark this answers is not in the document]
Comment (Rohitdhende) [reply package.json:17]: please revert this shit

File: src/components/Card/Card.react.jsx [hunk 2]
```diff
@@ -82,37 +100,38 @@ export default function CustomCard(props) {
...
-                  label={props.content.status}
+                  label={status}
                 />
               )}
             </div>
 
             <Typography
-              variant="body2"
-              className="line-clamp"
+              sx={{ fontSize: "0.8rem" }}
```
[the remark this answers is not in the document]
Comment (Rohitdhende) [reply src/components/Card/Card.react.jsx:133]: sure

**Review by anitsingh367 — APPROVED** [review]

File: src/components/Modal/ContributeModal.react.jsx [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+    setEmailValid(true);
+  };
+
+  useEffect(() => {
+    setOpen(props.isOpen);
+  }, [props.isOpen]);
+
+  return (
+    <Dialog open={open}>
+      <DialogTitle>Contribute</DialogTitle>
```
[the remark this answers is not in the document]
Comment (Rohitdhende) [reply src/components/Modal/ContributeModal.react.jsx]: Done fcker check it out

File: src/components/Card/Card.react.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Rohitdhende) [reply src/components/Card/Card.react.jsx:135]: hogya bhidu

File: src/components/Modal/ContributeModal.react.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Rohitdhende) [reply src/components/Modal/ContributeModal.react.jsx]: Done sir

**Pull request merged**